### ISSUE TITLE
Alive_bar() crashes when printing bytes (such as with click.echo)

### ISSUE THREAD

**Issue by jhonatan-lopes** [rsalmei/alive-progress#112]
Hi, I'm using click.echo to print messages as the bar is being displayed and I'm running into an error. Check out the example below:

```python
import click
import time
from alive_progress import alive_bar

# First example with standard print function, runs fine
# with alive_bar() as bar:
#     for i in range(100):
#         print(f'Printing iteration {i} with print')
#         time.sleep(0.05)
#         bar()

# Now, with click stdout
with alive_bar() as bar:
    for i in range(100):
        click.echo(f'Printing iteration {i} with click.echo')
        time.sleep(0.05)
        bar()
```
which gives me the error:

```
on 0: Traceback (most recent call last):
  File "venv/lib/python3.8/site-packages/alive_progress/core/progress.py", line 249, in __alive_bar
    yield bar
on 0: File "alive_progress_bug.py", line 15, in <module>
    click.echo(f'Packet no. {i}')
on 0: File "venv/lib/python3.8/site-packages/click/utils.py", line 298, in echo
    file.write(out)  # type: ignore
on 0: File "venv/lib/python3.8/site-packages/alive_progress/core/hook_manager.py", line 32, in flush
    write(stream, '\n')
on 0: File "venv/lib/python3.8/site-packages/alive_progress/core/hook_manager.py", line 45, in write
    nested = ''.join(line or ' ' * len(header) for line in buffer)
on 0: TypeError: sequence item 0: expected str instance, bytes found
on 0: During handling of the above exception, another exception occurred:
      
Traceback (most recent call last):
  File "alive_progress_bug.py", line 18, in <module>
    bar()
on 0: File "/usr/lib/python3.8/contextlib.py", line 131, in __exit__
    self.gen.throw(type, value, traceback)
on 0: File "venv/lib/python3.8/site-packages/alive_progress/core/progress.py", line 251, in __alive_bar
    stop_monitoring()
on 0: File "venv/lib/python3.8/site-packages/alive_progress/core/progress.py", line 151, in stop_monitoring
    hook_manager.uninstall()
on 0: File "venv/lib/python3.8/site-packages/alive_progress/core/hook_manager.py", line 70, in uninstall
    flush_buffers()
on 0: File "venv/lib/python3.8/site-packages/alive_progress/core/hook_manager.py", line 28, in flush_buffers
    flush(stream)
on 0: File "venv/lib/python3.8/site-packages/alive_progress/core/hook_manager.py", line 32, in flush
    write(stream, '\n')
on 0: File "venv/lib/python3.8/site-packages/alive_progress/core/hook_manager.py", line 45, in write
    nested = ''.join(line or ' ' * len(header) for line in buffer)
on 0: TypeError: sequence item 0: expected str instance, bytes found
```

**Comment by rsalmei** [comment]
Hey @jhonatan-lopes, good news! I'm glad to inform you it is done! I'm releasing it today, with a bunch of other things.
Thanks again for the PR!

**Comment by jhonatan-lopes** [comment]
Thanks for letting me know @rsalmei. It was my pleasure being able to contribute to such an amazing library.

**Comment by rsalmei** [comment]
Thank you 😄

**Comment by rsalmei** [comment]
Hey, it is released! #131 
Take a look at the new readme for details! 🎉
